src/icons/index.js

```diff
@@ -0,0 +1 @@
+export { default as EmptyStateArchiver } from './EmptyStateArchiver';
```

Comment:
sorry i might have missed but why do we need EmptyStateArchiver? Is that referenced from somewhere?

Reply:
This is just a good practice, it's recommended always have an index file that contains all the exports. So then, we could import components from the main folder. In this particular case imagine that we add more icons, we could use  named imports from /icons instead of adding multiple ones. 

**This:**
import { EmptyStateArchiver, Icon2, Icon3... } from './icons';

**Instead of:**
import EmptyStateArchiver from './icons/EmptyStateArchiver';
import Icon2 from './icons/icon2';
import ...